libs/platform/table/components/table-view-settings-dialog/grouping/grouping.component.html

```diff
@@ -43,9 +49,15 @@ <h4 fd-title [headerSize]="4">{{ 'platformTable.groupDialogHeader' | fdTranslate
 
         <ul fd-list [selection]="true">
             <li fd-list-group-header>
-                <span fd-list-title>{{ 'platformTable.groupDialogGroupByHeader' | fdTranslate }}</span>
+                <span fd-list-title id="group-dialog-group-by-header">{{
```

Comment:
I think this should also be `[attr.id]="groupOrderHeaderId"`

Reply:
good catch